pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
+
+- Replace all occurrences of a string in a file with another string, saving the original file with a given extension:
```

Comment:
Perhaps we could change `"saving the original file with a given extension"` to `"saving a back-up with a given extension"`?